src/style.css

```diff
@@ -0,0 +1,70 @@
+* {
+  box-sizing: border-box;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url("Avenir-Book.ttf");
+}
+
+body {
+  font-family: Avenir, Arial, sans-serif;
+  font-weight: 300;
+}
+
+.wrapper {
+  position: relative;
+  margin: 20px 0;
+}
+
+.decision {
```

Comment:
not sure about that, what is a decision, good when class names are simple and obvious